src/components/overall-statistics/HeroPictureSelect.vue

```diff
@@ -156,7 +156,7 @@ export default defineComponent({
     }
 
     function parsePicture(hero: HeroPick): string {
-      return getAsset(`heroes/${hero.heroId}.png`);
```

Comment:
Shall we maybe centralized this function so we don't have this logic replicated in multiple places?

Reply:
I'm not against it, but maybe in a separate PR. I'm still new here :sweat_smile: